components/log-viewer-webui/client/src/api/query.js

```diff
@@ -13,20 +13,21 @@ import axios from "axios";
  */
 
 /**
- * Submits a job to extract the split of an original file that contains a given log event. The file
- * is extracted as a CLP IR file.
+ * Submits a job to extract the stream that contains a given log event. The stream is extracted
+ * either as a CLP IR or a JSON line file.
```

Comment:
The official name of the spec is "JSON Lines": https://jsonlines.org/
```suggestion
 * either as a CLP IR or a JSON Lines file.
```